CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v1.7.0] - Unreleased Phase 6
 
 ### Added
+- Created CLI [Applet](https://github.com/signnow/SignNowNodeSDK/blob/master/bin/duplicate-template.js) for [Duplicate Template](https://signnow.github.io/SignNowNodeSDK/class/lib/template/index.js~Template.html#static-method-duplicate) feature.
```

Comment:
example https://github.com/signnow/SignNowNodeSDK/pull/74/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12. and move it to Change section pls